### PR TITLE
Fix race condition with Gitian in GitHub Actions when a Docker image is released.

### PR DESCRIPTION
**Pull request by y4ssi**

### CODE REVIEW

**Comment by yasserisa** [comment]
Tested ACK

File: .github/workflows/release-docker-hub.yaml [hunk 1]
```diff
@@ -12,10 +12,13 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       tags: ${{ steps.version_step.outputs.tags }}
+      zcash_version: ${{ steps.version_step.outputs.zcash_version }}
     steps:
     - id: version_step
       run:  |
         echo "tags=latest,${{ github.ref_name }},${{ github.sha }}" >> $GITHUB_OUTPUT
+        export zcash_version=$(echo ${{ github.ref_name }} | sed 's/v//g')
+        echo "zcash_version=$(echo $zcash_version)" >> $GITHUB_OUTPUT
```
Comment (str4d) [inline .github/workflows/release-docker-hub.yaml:21]: I think these two lines could be merged.

**Review by str4d — APPROVED** [review]
utACK

**Review by str4d — APPROVED** [review]
Re-utACK f7532003ae46b0f80a29844f46762d66365c7125

**Review by daira — COMMENTED** [review]
Post-hoc utACK

**Pull request merged**